### PR TITLE
[BIOMAGE-2197] - Create user and change password in other deployments

### PR DESCRIPTION
**Pull request by aerlaut**
- Create user in other deployment
- Change password in other deployment

https://biomage.atlassian.net/browse/BIOMAGE-2197

### CODE REVIEW

File: biomage/account/account.py [hunk 1]
```diff
@@ -34,99 +34,124 @@ def generate_password():
     )
 
 
-def create_account(full_name, email, userpool):
+def create_account(full_name, email, aws_profile, region, userpool):
```
Comment (cosa65) [inline biomage/account/account.py:37]: Is there anty chance you could take the region from the `aws_profile`?

File: biomage/account/account.py [hunk 2]
```diff
@@ -34,99 +34,124 @@ def generate_password():
...
+    cognito.admin_create_user(
+        UserPoolId=userpool,
+        Username=email,
+        MessageAction="SUPPRESS",
+        UserAttributes=[
+            {"Name": "email", "Value": email},
+            {"Name": "name", "Value": full_name},
+            {"Name": "email_verified", "Value": "true"},
+        ],
+    )
```
Comment (cosa65) [inline biomage/account/account.py:54]: Nice!

File: biomage/account/account.py [hunk 3]
```diff
@@ -34,99 +34,124 @@ def generate_password():
...
 
 @click.command()
 @click.option(
+    "-e",
     "--email",
     required=True,
     help="User email for the account to change the password in production.",
 )
 @click.option(
+    "-w",
```
Comment (cosa65) [inline biomage/account/account.py:65]: you could use `-P` here

File: biomage/account/account.py [hunk 4]
```diff
@@ -34,99 +34,124 @@ def generate_password():
...
+    help="Region the userpool is in.",
+)
+@click.option(
+    "-u",
     "--userpool",
     required=True,
     help="Userpool of the account to change.",
 )
-def change_password(email, password, userpool):
+def change_password(email, password, aws_profile, region, userpool):
```
Comment (cosa65) [inline biomage/account/account.py:92]: Is there anty chance you could take the region from the aws_profile?

File: biomage/account/account.py (hunk 1 above)
Comment (cosa65) [reply biomage/account/account.py:37]: Actually, I saw you only use it in `boto3.Session`, which I think doesn't really need to receive region, right?

Comment (aerlaut) [reply biomage/account/account.py]: yup, I think it'll use the default region set in `~/.aws/config`

File: biomage/account/account.py (hunk 3 above)
Comment (aerlaut) [reply biomage/account/account.py]: changed

File: biomage/account/account.py (hunk 4 above)
Comment (aerlaut) [reply biomage/account/account.py]: region removed, using default region in `~/.aws/config`

**Review by cosa65 — APPROVED** [review]

**Pull request merged**